tasks/blackduck-detect-task/ts/runner/DetectJarConfigurationRunner.ts

```diff
@@ -8,7 +8,7 @@ import {IJarConfiguration} from '../model/IJarConfiguration';
 import {logger} from '../DetectLogger';
 
 export class DetectJarConfigurationRunner extends DetectRunner {
-    private static readonly JAR_PREFIX = 'synopsys-detect-'
+    private static readonly JAR_PREFIX = 'blackduck-detect-'
```

Comment:
This will change to just detect- instead of blackduck-detect. Our detect jar does not contain blackduck in it.

Reply:
Dropped Blackduck prefix, thank you!